program/src/processor.rs

```diff
@@ -2770,7 +2710,7 @@ impl Processor {
             now_ts,
             referrer_mango_account_ai,
             limit,
-            is_luna_market,
+            is_market_closing,
```

Comment:
This will limit new bids for all closing markets to 10c, not just LUNA. Can't imagine this is desired, so will re-add is_luna_market and pass that instead